.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*        @mozilla/remote-settings-reviewers
```

Comment:
Looks like this team doesn't exist? What about https://github.com/orgs/mozilla-services/teams/syseng-pod?